src/pages/studyView/studyPageHeader/ActionButtons.tsx

```diff
@@ -64,8 +65,22 @@ export default class ActionButtons extends React.Component<ActionButtonsProps, {
         return 'Download clinical data for the selected cases';
     }
 
+    get virtualStudy(): JSX.Element | null {
```

Comment:
you should make this @computed so that it doesn't recompute unless it's observable dependecies have actually changed